web/server/parser/util.go

```diff
@@ -9,10 +9,17 @@ import (
 // parseTestFunctionDuration parses the duration in seconds as a float64
 // from a line of go test output that looks something like this:
 // --- PASS: TestOldSchool_PassesWithMessage (0.03 seconds)
+//
+// For Go1.5 and greater the test output looks something like this:
+// --- PASS: TestOldSchool_PassesWithMessage (0.03s)
 func parseTestFunctionDuration(line string) float64 {
 	line = strings.Replace(line, "(", "", 1)
 	fields := strings.Split(line, " ")
-	return parseDurationInSeconds(fields[3]+"s", 2)
+	timeStr := fields[3]+"s"
+	if strings.HasSuffix(timeStr, "s)s") {
```

Comment:
The < go 1.5 case is treated as the default and so in the previous line we have

```
timeStr := fields[3]+"s"
```
For < go 1.5 a line like
```
--- PASS: TestOldSchool_PassesWithMessage (0.03 seconds)
```

would have `fields[3] == 0.03` and `timeStr == 0.03s`

For >= go 1.5 with input like

```
--- PASS: TestOldSchool_PassesWithMessage (0.03s)
``` 

we'd have `fields[3] == 0.03s)` and `timeStr == 0.03s)s`